apps/src/code-studio/pd/application_dashboard/summary_table.jsx

```diff
@@ -39,7 +39,8 @@ export class SummaryTable extends React.Component {
       unreviewed: ApplicationDataPropType,
       waitlisted: ApplicationDataPropType,
     }),
-    path: PropTypes.string.isRequired
+    path: PropTypes.string.isRequired,
+    id: PropTypes.string
```

Comment:
Should we require this prop?

Reply:
I don't see the need. Normal elements don't require id.